ui/cockpit/src/modules/utils/config.js

```diff
@@ -22,6 +22,42 @@ const inProduction = process.env.NODE_ENV === "production";
 
 let config = {};
 
+let defaultConfig = {
```

Comment:
- defaultConfig can be `const`
- Let's move it into a separate json file. We have `json-loader` installed, so `import defaultConfig from  "./defaultConfig.json"` works.

Reply:
👍